Code/Legacy/CrySystem/XML/xml.cpp

```diff
@@ -320,7 +320,7 @@ void CXmlNode::setAttr(const char* key, int64 value)
 //////////////////////////////////////////////////////////////////////////
 void CXmlNode::setAttr(const char* key, uint64 value, bool useHexFormat)
 {
-    char str[32];
+    char str[32] = { 0 };
```

Comment:
nit: Rather than zero initializing the whole buffer, it can be preferable to write `'\0'` to the first element...

```
char str[32];
str[0] = '\0';
```

In this case it's not a huge deal but for large buffers you might not want to pay the cost of zeroing them before writing some other values to them.